client/public/index.html

```diff
@@ -1,17 +1,23 @@
 <!DOCTYPE html>
 <html dir="rtl" lang="ar">
-<head>
-  <meta charset="utf-8" />
-  <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta name="description" content="Rawdaty is a first platform to help you choose your children's kindergarten" />
+  <head>
+    <meta charset="utf-8" />
+    <!-- <link rel="icon" href="%PUBLIC_URL%/favicon.ico" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta
+      name="description"
+      content="Rawdaty is a first platform to help you choose your children's kindergarten"
+    />
 
-  <title>Rawdaty</title>
-</head>
+    <link
+      href="https://fonts.googleapis.com/css?family=Tajawal"
+      rel="stylesheet"
+    />
+    <title>Rawdaty</title>
+  </head>
 
-<body>
-  <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root"></div>
-</body>
-
-</html>
\ No newline at end of file
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
There's a problem with the indentation here, try to `pull origin main` and save the file again actually, there's no need to edit anything in this file except the description I told you to edit.